lib/components/Autosuggest/Autosuggest.tsx

```diff
@@ -700,4 +711,9 @@ export function Autosuggest<Value>({
       </Box>
     </Fragment>
   );
-}
+}) as <Value>(
```

Comment:
Just confirming this generic still works as intended from a consumer perspective?

Reply:
Yep, just double checked.